Src/Newtonsoft.Json/JsonConvert.cs

```diff
@@ -630,7 +630,7 @@ public static string SerializeObject(object? value, Type? type, JsonSerializerSe
         /// Serializes the specified object to a JSON string using formatting and <see cref="JsonSerializerSettings"/>.
         /// </summary>
         /// <param name="value">The object to serialize.</param>
-        /// <param name="formatting">Indicates how the output should be formatted.</param>
+        /// <param name="formatting">Indicates how the output should be formatted.  Note: this is will override any JsonSerializerSettings passed.</param>
```

Comment:
```suggestion
        /// <param name="formatting">Indicates how the output should be formatted.  Note: this will override any JsonSerializerSettings passed.</param>
```